conjur/util/credentials_from_keystore.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+
+"""
+CredentialsFromKeystore module
+
+This module holds the logic for saving user-specific credentials
+to the system's keyring when they login
+"""
+
+# Builtins
+import logging
+
+# Third party
+import keyring
+
+STORE_ATTRIBUTES = ["machine", "login_id", "api_key"]
```

Comment:
as in are you suggesting I change login_id -> login?

Reply:
No, just using some const or value from object, instead of a magic string like:
```suggestion
MACHINE="machine"
STORE_ATTRIBUTES = [MACHINE, "login_id", "api_key"]
```
and MACHINE being stored in conts or something